src/FunderMaps.Core/Entities/Building.cs

```diff
@@ -29,7 +29,7 @@ public Building()
         /// <summary>
         ///     Built year.
         /// </summary>
-        public DateTime BuiltYear { get; set; }
+        public DateTime? BuiltYear { get; set; }
```

Comment:
I noticed a recurring bug where built_year was null in our backend, see #346. Is this the fix for that issue?

Reply:
Jup, shoud have linked